{"name": "test_mis_audit", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "self = <scripts.test04_mis_audit.TestMisAudit object at 0x03B2A070>, title = 'test001-cd001', channel = '数据库'\n\n    def test_mis_audit(self, title=\"test001-cd001\", channel=\"数据库\"):\n        # 调用审核文章业务方法\n>       self.audit.page_mis_audit(title, channel)\n\nscripts\\test04_mis_audit.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npage\\page_mis_audit.py:64: in page_mis_audit\n    self.article_id = self.page_get_article_id()\npage\\page_mis_audit.py:43: in page_get_article_id\n    return self.base_get_text(page.mis_article_id)\nbase\\base.py:57: in base_get_text\n    log.info(\"正在对：{}元素获取文本操作！获取的文本值为：{}\".format(loc, self.base_find(loc).text))\nbase\\base.py:25: in base_find\n    poll_frequency=poll).until(lambda x: x.find_element(*loc)))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"20ab38c55793bfad715486a411c3f732\")>\nmethod = <function Base.base_find.<locals>.<lambda> at 0x030011E0>, message = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value is not False.\"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\nc:\\program files (x86)\\python\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:80: TimeoutException"}, "attachments": [{"name": "log", "source": "10a0bc6c-a0e1-4e60-8e12-4431acb89c03-attachment.txt", "type": "text/plain"}], "start": 1594137090151, "stop": 1594137127629, "uuid": "6221b89d-5bcd-4bb4-9f7e-0f540cad2353", "historyId": "9882f7a913a37e1c698f7c88bfc6c7ef", "testCaseId": "a9c2bf081396218a64bd35d9487692ca", "fullName": "scripts.test04_mis_audit.TestMisAudit#test_mis_audit", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test04_mis_audit"}, {"name": "subSuite", "value": "TestMisAudit"}, {"name": "host", "value": "DESKTOP-VRLRK3E"}, {"name": "thread", "value": "7684-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test04_mis_audit"}]}